plugin/githubinator.vim

```diff
@@ -89,13 +87,18 @@ function! s:github_copy_url()
         return
     endif
 
-    echom 'Githubinator: URL copied to clipboard.'
+    echom 'Githubinator: URL copied to clipboard: ' . l:final_url
```

Comment:
I made the URL is output on command line window to confirm it is what I want to copy.

Reply:
When using `ghc`, a trailing newline is also there. Can you fix this so that we can get this merged? thanks

Reply:
> a trailing newline is also there.

It is not appeared on my environment.
![2019-03-20-234150_1145x120_scrot](https://user-images.githubusercontent.com/1330045/54693332-c4634480-4b69-11e9-8b49-8727d0f33792.png)

You mean that there is newline after `L66-L70` ?
Could you give me screenshot ?

Reply:
yes, could you try pasting it somewhere? It'll be evident.

Reply:
Oops, I realize it :sweat_smile: 
I add a new commit to fix it !

Reply:
The reason why there is a newline at tail is `xsel`.
`echo abcde | xsel -b` reproduces !

Also this PR does not caused this behaviour.

Reply:
I fixed it.
Please check !
https://github.com/danishprakash/vim-githubinator/pull/14/commits/afc3084cac7e1a98f9cb2a405aa709807a878664